### DISCUSSION TITLE
DateTime filter not working when using a join query, and the field name is the same as "created_at."

### DISCUSSION THREAD

**Discussion by jenishprajapati-kombee** [Power-Components/livewire-powergrid#1293] [BUG]
I am also adding a prefix table name with a filed name but it's not working.

My code: 

Query: 
![image](https://github.com/Power-Components/livewire-powergrid/assets/67471303/2ce4ac0d-c13a-4c51-b149-18aadc920a18)

Filter: 
![image](https://github.com/Power-Components/livewire-powergrid/assets/67471303/9727f1fc-755f-4277-9532-42a75e098e38)

**Comment by luanfreitasdev** [comment]
Hey @jenishprajapati-easternts, I recently sent a fix for this, but haven't released a release yet.
Can you test if this works if you install dev-main? 

Thanks

**Reply by luanfreitasdev** [reply]
https://github.com/Power-Components/livewire-powergrid/pull/1286
